src/shared/features/Inbox/Inbox.tsx

```diff
@@ -1,15 +1,16 @@
 import React, { useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
-import { Container, Grid, Card, CardHeader, CardContent } from '@material-ui/core';
+import {Container, Grid, Card, CardHeader, CardContent, Box} from '@material-ui/core';
 import MailIcon from '@material-ui/icons/Mail';
 import DraftsIcon from '@material-ui/icons/Drafts';
-import { Link } from 'react-router-dom';
 
+import { Link } from 'react-router-dom';
 import { Post } from 'shared/models/post.model';
 
 import './Inbox.css';
 
+// todo: question: why here versus Inbox.css ?
```

Comment:
Material UI for React code examples always have these embedded classes that seem to be their own system independent of a .css file. I brought them over for the sake of getting things set up quickly, but I think styling belongs in the .css files (and only in the css files) and we should remove the embedded styles as we build the app. 

The system shown here hooks into theming and other stuff I haven't researched yet, though, so maybe there's merit to keeping it? We should look into it before totally dismissing it.

Reply:
I definitely agree with more research.